pkg/dir.props

```diff
@@ -58,6 +58,24 @@
     <HostFullVersion>$(HostVersion)$(PreReleaseSuffix)</HostFullVersion>
   </PropertyGroup>
 
+  <Import Project="$(ToolsDir)Build.Common.props" Condition="Exists('$(ToolsDir)Build.Common.props')" />
+
+  <ItemGroup>
+    <NuGetSource Include="https:%2F%2Fdotnet.myget.org/F/dotnet-core-dev-eng/api/v3/index.json" />
```

Comment:
This is our feed for our different branch.  We're only putting the stuff from the dev/eng effort in this feed.